proofs/tree.go

```diff
@@ -54,6 +54,12 @@ import (
 
 const DefaultSaltsLengthSuffix = "Length"
 
+/*
```

Comment:
This should start with something like:
// TreeOptions struct contains options that describe the tree generation


> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.
https://golang.org/doc/effective_go.html#commentary

Reply:
Most Go code I've seen uses // for function and type comments.